### PR TITLE
Add "Pause All"/"Resume All" button for devices #6530

### PR DESCRIPTION
**Pull request by MikolajTwarog**
I added the pause all/resume all button for devices.

### CODE REVIEW

**Comment by MikolajTwarog** [comment]
Capture
Capture2

**Comment by AudriusButkevicius** [comment]
Presumably it looks the same as for folders?

**Comment by MikolajTwarog** [comment]
Yes, it is the same.

File: gui/default/index.html [hunk 1]
```diff
@@ -788,6 +788,12 @@ <h4 class="panel-title">
         </div>
         <div class="form-group">
           <span class="pull-right">
+            <button type="button" class="btn btn-sm btn-default" ng-click="setAllDevicesPause(true)" ng-if="isAtleastOneDevicePausedStateSetTo(false)">
+             <span class="fas fa-pause"></span>&nbsp;<span translate>Pause All</span>
+            </button>
+            <button type="button" class="btn btn-sm btn-default" ng-click="setAllDevicesPause(false)" ng-if="isAtleastOneDevicePausedStateSetTo(true)">
+             <span class="fas fa-play"></span>&nbsp;<span translate>Resume All</span>
```
Comment (imsodin) [inline gui/default/index.html:795]: minor nit: please use the same indentation of two spaces.

Comment (MikolajTwarog) [reply gui/default/index.html]: Sorry. I did't notice it. It is fixed now.

**Pull request merged**